geode-core/src/main/java/org/apache/geode/internal/cache/DistributedRegion.java

```diff
@@ -1298,6 +1299,24 @@ private void synchronizeWith(InternalDistributedMember target, VersionSource ver
     op.synchronizeWith(target, versionMember, lostMember);
   }
 
+
+  public void setRegionSynchronizeScheduled(VersionSource lostMemberVersionID) {
```

Comment:
Javadocs for the new public methods you've added in DistributedRegion and other places would help me understand this code more easily.  The long variable and method names would help more if I knew the context of the problem these methods are solving.

Reply:
Will add more java docs for these method.

Reply:
I have added more comments on the newly introduced methods.